models/iris_gps_height/iris_gps_height.sdf

```diff
@@ -0,0 +1,618 @@
+<sdf version='1.7'>
```

Comment:
I guess same problem as before, but since we don't officially support sdf version 1.7 I think it is better we keep it as 1.6